modules/xmpp/moderator.js

```diff
@@ -210,11 +210,12 @@ Moderator.prototype.createConferenceIq = function() {
                 value: this.options.conference.minBitrate
             }).up();
     }
-    if (this.options.conference.openSctp !== undefined) {
+    if (this.options.conference.openBridgeChannel !== undefined) {
```

Comment:
This is a bit "ugly" since, in `moderator.js`,  I should replicate all this code already present in `JitsiConference.js`:

```js
    let openBridgeChannel;

    switch (this.options.config.openBridgeChannel) {
    case 'datachannel':
    case true:
    case undefined:
        openBridgeChannel = 'datachannel';
        break;
    case 'websocket':
        openBridgeChannel = 'websocket';
        break;
    }

    // NOTE: Edge does not yet implement DataChannel, so force WebSocket.
    if (RTCBrowserType.isEdge() && openBridgeChannel === 'datachannel') {
        openBridgeChannel = 'websocket';
    }
```